### PR TITLE
Add discussion of saving queries to views to basic SQL

### PR DESCRIPTION
**Pull request by ethanwhite**

### CODE REVIEW

File: materials/sql-basic-queries.md [hunk 1]
```diff
@@ -55,8 +55,24 @@ SELECT species_id, ROUND(hindfoot_length/1000.0, 2)
...
+
+### Saving queries for future use
+
+* Views save queries to run again.
+* Create them by using `Create View` in the `View` menu, or by adding `CREATE
+  VIEW *viewname* AS` to the beginning of a query.
+
+```
+CREATE VIEW species_count AS
+SELECT species_id, plot_id MIN(weight), MAX(weight), AVG(weight)
```
Comment (brymz) [inline materials/sql-basic-queries.md]: `..., plot_id, MIN(weight), ...`

File: materials/sql-basic-queries.md [hunk 2]
```diff
@@ -55,8 +55,24 @@ SELECT species_id, ROUND(hindfoot_length/1000.0, 2)
...
+> minute.
+
+### Saving queries for future use
+
+* Views save queries to run again.
+* Create them by using `Create View` in the `View` menu, or by adding `CREATE
+  VIEW *viewname* AS` to the beginning of a query.
+
+```
+CREATE VIEW species_count AS
```
Comment (brymz) [inline materials/sql-basic-queries.md]: Does this example come too soon? In the future, you won't have given aggregation lecture.

Comment (ethanwhite) [inline materials/sql-basic-queries.md]: Yes, totally. I hadn't originally written this up later and then realized they needed it for the assignment. Will fix shortly.

**Comment by ethanwhite** [comment]
Fixed. Thanks.

**Comment by brymz** [comment]
It looks like you modified the original commit instead of making a second commit with changes. Can you tell me how you did that? It definitely strikes me as a preferred method for minor changes.

**Comment by ethanwhite** [comment]
```
git add filename
git commit --amend [you can either just save the file with the same commit message or update the message]
git push -f origin branchname
```

**Pull request merged**